src/contexts/GustoApiProvider/GustoProviderWithUIAdapter.tsx

```diff
@@ -0,0 +1,39 @@
+import type React from 'react'
```

Comment:
My thought here is that the GustoProviderWithUIAdapter would be the more common one so maybe this should just be GustoProvider? my reasoning there would be so that the power user needs to deal with the weirder name 😂 so if you don't care about supplying your own components and are fine rolling with our stuff you just bring in the intuitive `GustoProvider` component. If you want to do some advanced usage you bring in `GustoProviderWithoutUIAdapter` or whatever we want to call it naming is hard lol